src/main/webapp/createPlaceGuide.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset = "UTF-8">
+    <title>Create Place Guide</title>
+    <style>
+      #imageDiv {
+        width: 300px;
+        min-height: 100px;
+        border: 2px solid #dddddd;
+
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        font-weight: bold;
+        color: #cccccc;
+      }
+
+      #imagePreview {
+        display: none;
+        width: 100%;
+      }
+    </style>
+    <script src = "setupCreatePlaceGuide.js"></script>
+</head>
+<body onload = "setUpCreatePlaceGuidePage()">
+<div id="createPlaceGuide">
+    <form method = "POST" id = "formId" enctype = "multipart/form-data">
+        <input type="hidden" id="id" name="id">
+        <br>
+        <label for = "name">Name:</label><br>
+        <input type = "text" name = "name" id = "name" required>
+        <br>
+        <label for = "audioKey">Upload audio file</label>
+        <input type = "file" name = "audioKey" id = "audioKey" required>
+        <br>
+        <audio id = "audioPlayer" controls>
+            <source src="" type="audio/mpeg">
+        </audio>
+        <br>
+        <label for = "deleteAudio">Delete previous audio file</label>
+        <input type = "checkbox" name = "deleteAudio" value = "delete" id = "deleteAudio">
+        <br>
+        <label for = "imageKey">Upload image</label>
+        <input type = "file" name = "imageKey" id = "imageKey">
+        <div id = "imageDiv">
+        <br>
+        <img src = "" alt = "image preview" id = "imagePreview">
+        <br>
+        </div>
+        <label for = "deleteImage">Delete previous image</label>
+        <input type = "checkbox" name = "deleteImage" value = "delete" id = "deleteImage">
+        <br>
+        <label for = "placeId">place id : </label><br>
+        <input type = "text" name = "placeId" id = "placeId" readonly><br>
+        <br>
+        <label for = "description">Description:</label><br>
+        <textarea name = "description" rows = "4" cols = "50" id = "description"></textarea>
+        <br>
+        <label for = "length">Amount of time needed:</label><br>
+        <input type = "text" name = "length" id = "length"><br>
+        <label for = "isPublic">Place Guide Status:</label>
+        <select name = "isPublic" id = "isPublic">
+            <option value = "public">public</option>
+            <option value = "private">private</option>
+        </select>
+        <br>
+        
+        <fieldset>
+            <legend>Coordinate: </legend>
+            <label for = "latitude">latitude : </label><br>
+            <input type = "text" name = "latitude" id = "latitude" required><br>
+            <label for = "longitude">longitude : </label><br>
+            <input type = "text" name = "longitude" id = "longitude" required><br>
+            <br>        
```

Comment:
Since required and readonly cannot exist in the same tag, I have changed it with :
`required onkeypress="return false;"`
This way, the user still can "type" on the input form, but nothing will appear on the input form. The text cursor will also still appear when the user tries to type something on the input form. The input form will still be required.